examples/app_commands/basic.py

```diff
@@ -68,4 +68,30 @@ async def joined(interaction: discord.Interaction, member: Optional[discord.Memb
     await interaction.response.send_message(f'{member} joined in {member.joined_at}')
 
 
+# A Context Menu is an app command that can be run on a member or on a message by rightclicking.
+# It always takes an interaction as its first parameter and a Member, Message or Union of both as its second parameter.
+
+# This context menu only works for members
+@client.tree.context_menu(name='Joindate')
+async def show_join_date(interaction: discord.Interaction, member: discord.Member):
+    await interaction.response.send_message(f'{member} joined in {member.joined_at}')
```

Comment:
```suggestion
    # The format_dt function formats the date time into a human readable representation in the official client
    await interaction.response.send_message(f'{member} joined in {discord.utils.format_dt(member.joined_at)}')
```

Might as well turn this into a human readable date.

Reply:
Should I also change this in the /joined command?

Reply:
Sure.